Cards/MadGraph5_aMCatNLO/DY/DYto2L-2Jets_MLL-50_2J_amcatnloFXFX-pythia8/DYto2L-2Jets_MLL-50_2J_amcatnloFXFX-pythia8.json~

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Also json~

Reply:
sorry. missed that one as well. removed.